dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+- package-ecosystem: cargo
+  directory: "/"
+  schedule:
+    interval: "daily"
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+  allow:
+  - dependency-type: direct
```

Comment:
Why is this necessary? I couldn't figure out what's the default for this from the docs, but we don't commit Cargo.lock, so there shouldn't be any difference, no?

Reply:
Should be no difference indeed, based on only having a `Cargo.toml` file